lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -41,10 +41,11 @@
             EmailFieldView: FieldViews.TextFieldView.extend({
                 fieldTemplate: field_text_account_template,
                 successMessage: function () {
-                    return this.indicators.success + StringUtils.interpolate(
+                    return HtmlUtils.joinHtml(
+                        this.indicators.success, StringUtils.interpolate(
```

Comment:
I take this back, sort of.  I'm confused as to whether `this.indicators.success` contains HTML or not.  

Was this actually giving a validation error as it was, or was there just a defect here?